static/js/script.js

```diff
@@ -1,4 +1,12 @@
- //modal close button
+function addAnchorTags() {
+  anchors.options = {
+    visible: 'touch'
+  }
+
+  anchors.add('#docsContent h2, #docsContent h3');
```

Comment:
I don't think you should stop at h3, why not go all the way down to 6? I didn't notice this before.

Reply:
@mistyhacks Ah, good call. I wrongly assumed that the header sizes stopped at h3. A simple `git grep` showed that that is quite incorrect 😄